neuro_san/http_sidecar/handlers/connectivity_handler.py

```diff
@@ -32,7 +32,8 @@ async def get(self, agent_name: str):
         if not update_done:
             return
 
-        if not self.agent_policy.allow(agent_name):
+        service: AsyncAgentService = self.agent_policy.allow(agent_name)
+        if service is None:
```

Comment:
Our policy will now provide an instance of AsyncAgentService for this particular agent,
if servicing of this agent is allowed.